03-task-manager/starter/QuizAnswers2.txt

```diff
@@ -0,0 +1,16 @@
+1. In this lesson, you created a middleware function called asyncWrapper. Why?
+
+With asynchronous operations, using try-catch blocks is handy, but it tends to get repetitive.
+So, a slicker fix is setting up a middleware function to wrap around the controllers. 
+This way, we dodge the redundancy and keep things neat and tidy.
```

Comment:
Also makes code more maintainable because you have a centralized few files to control error behavior.

Love the custom error message below 😄 Nice job